pkg/scheduler/scheduler.go

```diff
@@ -507,26 +510,58 @@ func (sched *Scheduler) assume(assumed *v1.Pod, host string) error {
 
 // bind binds a pod to a given node defined in a binding object.  We expect this to run asynchronously, so we
 // handle binding metrics internally.
-func (sched *Scheduler) bind(ctx context.Context, assumed *v1.Pod, targetNode string, state *framework.CycleState) error {
-	bindingStart := time.Now()
+func (sched *Scheduler) bind(ctx context.Context, assumed *v1.Pod, targetNode string, state *framework.CycleState) (err error) {
```

Comment:
This gives precedence to the extenders to bind the pod, the original code checked the framework plugins first. I think this makes sense, but it would be good to have a comment clarifying the order.

Reply:
Indeed. Once we add the default plugin, we have to check extenders first for backwards compatibility.